public/locales/en/translation.json

```diff
@@ -1,6 +1,7 @@
 {
   "title": "Project Title",
   "nav": {
+    "dropdown": [],
```

Comment:
Did you mean to have an empty array here or should it have the dropdownOne info in it

Reply:
The empty array is there on purpose, it was a placeholder. The array is now filled in the next PR.